example/README.md

```diff
@@ -1,4 +1,31 @@
+# BringAuto Package Example Context Directory
 
-# BringAuto Package example context directory
+This example context directory is used for testing and guidance.
 
-Example context directory used for testing and guidance.
\ No newline at end of file
+## Helper Scripts
+
+- **`add_docker_to_matrix.sh`** - Adds a new Docker image to all package JSON files in the context directory. In the example, it adds the `ubuntu2310` Docker image to all package JSON files.
+
+- **`change_docker_name.sh`** - Changes the Docker image name in all package JSON files in the context directory. In the example, it changes `fleet-os` to `fleet-os-2`.
+
+## Package JSON Structure
+
+- **`Env`**: Environment variables used for the project.
```

Comment:
There soud be a documentation for a .json format. Look at doc/ dicrectory.

Reply:
Update/specify existing documentation pls. DO not duplicate.

Reply:
Done by Ondřej Vala.